mods/d2k/mod.yaml

```diff
@@ -13,23 +13,23 @@ FileSystem: ContentInstallerFileSystem
 		$d2k: d2k
 		^EngineDir|mods/common: common
 		~^SupportDir|Content/d2k/v3/: content
-		d2k|scripts
 		common|scripts
+		d2k|scripts
 		d2k|uibits
 	ContentPackages:
 		content|SOUND.RS
 		content|GAMESFX
-		d2k|bits
+		d2k|bits    # This needs to be here so its contents override DATA.R16 that is inside `content`.
```

Comment:
This comment should be on RA and TD, but probably not needed here (since DATA.R16 isn't a package!)